week-4/B-objects-get-set/README.md

```diff
@@ -1,4 +1,52 @@
-### Goals
-- reading object properties using the *dot notation*: `console.log(myObject.name)`
-- setting object properties: `myObject.name = "John"`
-- simpler exercises on this as well, to get them used to reading/writing on objects
\ No newline at end of file
+### Getting the value of an object's property
+
+Let's take one of the objects we looked at earlier..
+
+```js
+var laptop = {
+    brand: "Lenovo",
+    screenSize: 13,
+    isTouchscreen: true
+};
+```
+
+> Try to `console.log(laptop)`. The output might depend on your environment!
+
+To find out the value of an object's property, you can use the dot notation..
+
+```js
+console.log(laptop.brand);

```

Comment:
This section might be better to follow along if the examples are in the node console (rather than console.log from a file)

Reply:
That makes sense - when I show this in front of the class I'll make sure to use Node.

I thought I would still leave them in here as `console.log` as that gives a better indication of how it's being used.

Alternatively, I could use a variable:

```js
var brand = laptop.brand;
```

Reply:
For future reference, in previous exercises I've used this format :

```sh
$ node
> 1 + 2
3
> "hello"
'hello'
> var greeting = "hello"
undefined
> greeting
'hello'
> console.log(greeting)
hello
undefined
> .exit
$
```

Example: https://github.com/CodeYourFuture/js-exercises/tree/master/week-2/F-logical-operators